packages/fast-layouts-react/src/page/page.props.ts

```diff
@@ -1,4 +1,5 @@
 import React from "react";
+import { GridDisplay } from "../grid/grid.props";
```

Comment:
@JamesBurnside @nicholasrice I wonder if we want to move this definition *out* from Grid? This change makes the interface a bit more of a utility specific to the CSS grid property name. It no longer is specific to the grid component implementation, and I think we should likely move it as part of this change.

Reply:
I'd be open to moving within this package, but I think it's a bit fragmented of an implementation to put into web-utilities or other packages.

Reply:
I could move into fast-layouts-react/utilities into its own file of something like "grid-display.ts" or "layout-styles.ts"

Reply:
Yes, this is what I'm referring to. I think we want it within this package, just not in the grid.props.ts file.

I think for now it's likely fine to even just have an index file in a utils folder until we need to add more:
```
fast-layouts-react/utilities/index.ts
```